Add TH localization for Soundy Music Player

OatMealXXII opened the pull request:
Thai localization translated by Thai native

OatMeal08092552 committed:
Add TH locales

OatMeal08092552 committed:
Fix some wrong in TH

OatMealXXII commented:
I already test all that work!

idMJA commented:
> I already test all that work!

what is the translation of this?

```ts
powered_by: ({ provider }) => `Powered by ${provider}`
```

OatMealXXII commented:
> > I already test all that work!
> 
> what is the translation of this?
> 
> ```ts
> powered_by: ({ provider }) => `Powered by ${provider}`
> ```

**This:**  powered_by: ({ provider }) => Powered by ${provider}``

**Translated:**  powered_by: ({ provider }) => โดย ${provider}

idMJA committed:
fix: update Thai locale for improved readability and consistency

chore: remove unnecessary entries from .gitignore and update package.json dependencies

idMJA commented:
thanks
